docs/source/architecture.rst

```diff
@@ -151,7 +151,7 @@ Configurable classes.
 .. seealso::
 
     - :doc:`customizing`
-    - :ref:`jinja:filters`
+    - `More on Jinja Filters`_
```

Comment:
updated link. this used `:ref:` before which is a local page link. I also fixed the URL to the jinja template page